packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -2604,12 +2755,59 @@ describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb stopgaps', () => {
       expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
-                                        // *** WARNING ***
-                                        // 
-                                        // These models are using a row level TTL setting defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/row-level-ttl
-                                        // - Model "ttl_test"
-                                        // 
-                              `)
+                                                                                                                                        // *** WARNING ***
+                                                                                                                                        // 
+                                                                                                                                        // These models are using a row level TTL setting defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/row-level-ttl
+                                                                                                                                        // - Model "ttl_test"
+                                                                                                                                        // 
+                                                                                                      `)
+      expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+  })
+
+  describe('warning 36/5 - comments found: models & fields', () => {
+    const stopGap = 'comments'
+    const warningCode = 36
+    const variant = 5
```

Comment:
What does the variant actually identify here? 
It's weird that the variants are spread across different databases, isn't it?
Isn't this variant 1 for Cockroach?

Reply:
confused me a tiny bit, but doesn't matter much as long as the test is there.